enhancements/sig-policy/128-resource-selector/README.md

```diff
@@ -0,0 +1,142 @@
+`LabelSelector` since that can be handled by the Kubernetes API. The `namespaceSelector` has
+`include` and `exclude` filters for filtering by name, but such a feature requires that it run in
+its own `Namespace` reconciler, which would be unwieldy with a `resourceSelector` where the `Kind`
+is arbitrary.
+
+In a YAML manifest, this new field would appear as follows:
+
+```yaml
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: resource-selector-policy
+spec:
+  remediationAction: inform
+  namespaceSelector:        # <---- Existing namespaceSelector
+    exclude: []             #
+    include: []             #
+    matchLabels: {}         #
+    matchExpressions: []    #
+  object-templates:
+    - complianceType: musthave
+      resourceSelector:         # <---- New resourceSelector
+        matchLabels: {}         #
+        matchExpressions: []    #
+      objectDefinition:
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          labels:
+            selected: true
+    - ...
+```
+
+The `config-policy-controller` would list the provided `Kind` across all namespaces using the
+provided `resourceSelector` and iterate over each maching object in each namespace to determine
+compliance. Namespaces would be determined from either the namespace defined in the
+`objectDefinition` or the
+
+### Risks and Mitigation
+
+- The implementation could increase load on the Kubernetes API.
+  - Depending on how it's implemented, I wouldn't think the load would be any greater than if
+    `object-templates-raw` were in play. Ideally the initial `List` request would populate the cache
+    and any subsequent `Get` (if any) would fetch from the cache.
+- Adding a `resourceSelector` could enable enforcement where enforcement wasn't previously
+  available, which could have unintended consequences for users.
+  - One possibility here would be to not enable enforcement, but I think there would be more demand
+    for enforcement than not. And a change would need to be made to the policy in order to enable
+    it, which hopefully users would test beforehand.
+
+### Open Questions
+
+1. Should `remediationAction: enforce` be allowed?
+   - Some users may rely on the current behavior that an unnamed `objectDefinition` can only be
+     `inform` and may cause unexpected results when a `resourceSelector` is newly applied to a
+     policy they didn't realize was enforced.
+2. Should the `resourceSelector` be able to determine the namespace if the namespace is not
+   provided?
+   - Currently if the namespace is not provided in the `objectDefinition` and the
+     `namespaceSelector` returns no results, no objects are returned. I think this behavior should
+     be preserved since it would follow the behaviors of tools like `kubectl`, but I could be
+     convinced otherwise.
+3. Should the `resourceSelector` have include/exclude name filtering?
```

Comment:
I think the way you have it designed with the namespaceSelector feeding into the resourceSelector is good. No need for further filtering.